src/main/java/net/sf/jabref/gui/keyboard/KeyBinding.java

```diff
@@ -13,6 +13,7 @@
     BACK("Back", Localization.lang("Back"), "alt LEFT"),
     CLEANUP(
             "Cleanup", Localization.lang("Cleanup entries"), "F8"),
+    CHECK_INTEGRITY("Check integrity", Localization.lang("Check integrity"), "ctrl F8"),
```

Comment:
The thing I rember was about the keybinding **category** where we should use `Localization.menuTitle` as you wrote in https://github.com/JabRef/jabref/pull/1390#issuecomment-238339367. This affects only the javafx branch so far. Other than that I don't remeber a discussion about the menu entries in the hotkey dialog. :thinking: